fission/log/log.go

```diff
@@ -32,6 +32,10 @@ func Fatal(msg interface{}) {
 }
 
 func Warn(msg interface{}) {
+	os.Stderr.WriteString(fmt.Sprintf("[WARNING] %v\n", msg))
+}
+
+func Log(msg interface{}) {
```

Comment:
Might be better to name this `Info`, which is more in line with naming in common log libraries

Reply:
Done